src/spock_relcache.c

```
@@ -95,74 +96,74 @@ spock_relation_open(uint32 remoteid, LOCKMODE lockmode)
+			pfree(entry->delta_apply_functions);
+		{
+			MemoryContext oldcontext;
+
+			oldcontext = MemoryContextSwitchTo(CacheMemoryContext);
+			entry->delta_apply_functions =
+				palloc0(desc->natts * sizeof(Oid));
+			MemoryContextSwitchTo(oldcontext);
+		}
+		entry->has_delta_columns = false;
 
 		for (i = 0; i < entry->natts; i++)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If pg_attribute_printf is a macro then please configure it.

(unknownMacro)